example/README.md

```diff
@@ -0,0 +1,23 @@
+An simple example of a Dart package using [json_serializable].
+
+* `lib/example.dart`: A library configured with annotations – 
+  `JsonSerializable` and `JsonLiteral` – to enable code generation.
+
+  * Note: the annotations are defined in `package:json_annotation`.
+    This is the only package required in the `dependencies` section of your
+    `pubspec.yaml`.
+
+* `tool/`: Contains the code run during development to create and update
+  generated code.
+
+  * `build_actions.dart`: A convention when using `package:build` to 
```

Comment:
We actually should change this convention - having separate scripts means they have separate asset graphs which isn't what you want.

Instead you could add a flag to change whether it calls build/watch?

Reply:
Then I have to add args parsing, etc to the example.

I'd rather have https://github.com/dart-lang/build/issues/718